lib/widget/confirmation_input.dart

```diff
@@ -39,6 +39,15 @@ class ConfirmationInput extends StatefulWidget
           _controller.enableCursor = Utils.getBool(newValue, fallback: true),
       'length': (newValue) =>
           _controller.length = Utils.getInt(newValue, fallback: 4),
+      'width': (value) => _controller.fieldWidth = Utils.optionalDouble(value),
```

Comment:
name the variables what they are, fieldWidth/fieldHeight, since width/height normally apply to the root widget.

Reply:
I have done this but later I set it as width and height instead.
Okay, I'll change it back to `fieldWidth` and `fieldHeight`.